src/ioctls/vm.rs

```diff
@@ -733,4 +772,18 @@ mod tests {
         let mut kvi: kvm_bindings::kvm_vcpu_init = kvm_bindings::kvm_vcpu_init::default();
         assert!(vm.get_preferred_target(&mut kvi).is_ok());
     }
+
+    #[test]
+    #[cfg(any(
+        target_arch = "x86",
+        target_arch = "x86_64",
+        target_arch = "arm",
+        target_arch = "aarch64"
+    ))]
+    fn test_signal_msi_failure() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let msi = kvm_msi::default();
+        assert!(vm.signal_msi(msi).is_err());
```

Comment:
Add an explanation to why we expect this to be an error.

Reply:
Yes will do.

Reply:
done